### DISCUSSION TITLE
Prisma-based database engine

### DISCUSSION THREAD

**Discussion by chientrm** [prisma/prisma#16579] [Q&A] [answered]
# Introduction

From what I observed, `prisma` build an abstract layer upon common database engines like `MySQL`, `MSSQL`, ...

`prisma` introduce script-level concepts that fit most SDLC use cases.

# Propose
Build a `prisma-based` database engine that leverages no other common database engines but runs directly on `NodeJS`.

# Benefits

- Help reduce the overcomplexity of CI/CD management.
- Experiment with new concepts that can never be built upon current engines.

**Comment by pantharshit00** [comment, marked as answer]
Hello @chientrm,

It would be great if you can open this ticket as a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=). That way it can be tracked and people working on the core library will able to comment there.

**Reply by chientrm** [reply]
> Hello @chientrm,
> 
> It would be great if you can open this ticket as a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=). That way it can be tracked and people working on the core library will able to comment there.

Opened a feature request at #16590

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
